### PR TITLE
fix: update FAQ link and remove need help

### PR DESCRIPTION
**Pull request by franckwei**
### Ticket ID

https://app.clickup.com/t/2645698/DEV-9813

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-9813 Bug: change FAQ link](https://app.clickup.com/t/2645698/DEV-9813)

File: components/AppFooter.vue [hunk 1]
```diff
@@ -20,18 +20,12 @@ const providers: ProviderName[] = ['twitter', 'discord', 'github']
           {{ $formatMessage('footer_docs_text') }}
         </a>
         <a
-          href="https://lukso.network/faq"
+          href="https://support.lukso.network/"
           class="nav-apax-12-medium-uppercase text-purple-41 hover:text-purple-31 hover:underline"
           target="_blank"
         >
           {{ $formatMessage('footer_faq_text') }}
```
Comment (Hugoo) [inline components/AppFooter.vue:27]: Ill put the "Need help?" text here rather than the FAQ text

Comment (Hugoo) [reply components/AppFooter.vue:27]: `footer_need_help_text`

Comment (franckwei) [reply components/AppFooter.vue]: done

**Review by Hugoo — APPROVED** [review]

**Pull request merged**